llvm/include/llvm/DebugInfo/DIContext.h

```diff
@@ -71,8 +78,8 @@ struct DILineInfo {
     OS << "Line info: ";
     if (FileName != BadString)
       OS << "file '" << FileName << "', ";
-    if (FunctionName != BadString)
-      OS << "function '" << FunctionName << "', ";
+    if (SymbolTableFunctionName != BadString)
+      OS << "function '" << SymbolTableFunctionName << "', ";
```

Comment:
FunctionName no longer exists. This change was made so that the code compiles. 
Why SymbolTableFunctionName and not Linkage or ShortFunctionName? Well I don't think it matters we are using DIPrinter.cpp to get this information.